<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenQuestionsWindow</class>
 <widget class="QMainWindow" name="OpenQuestionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1943</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>~Experiment~</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{background: white;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1710</width>
      <height>984</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>320</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>200</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QTextEdit" name="instructionsBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>AvenirNext LT Pro Regular</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit{border: 1px solid white;}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="questionBox2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1500</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2500</width>
         <height>320</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>AvenirNext LT Pro Regular</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="title">
        <string>In this session, how did you estimate the accuracy (your strategy)? E.g., any visual features or patterns you looked for?</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>1452</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QTextEdit" name="answer2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>AvenirNext LT Pro Regular</family>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="questionBox1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1500</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2500</width>
         <height>320</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>AvenirNext LT Pro Regular</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Any other comments about this session? E.g., if you have any difficulties?</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>49</y>
          <width>1452</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QTextEdit" name="answer1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>AvenirNext LT Pro Regular</family>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft|Qt::AlignTop">
      <widget class="QPushButton" name="nextButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>AvenirNext LT Pro Regular</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
